projects/eval-adxl314z/Makefile

```diff
@@ -0,0 +1,10 @@
+# Select the example you want to enable by chossing y for enabling and n for disabling
+DUMMY_EXAMPLE = y
+IIO_EXAMPLE = n
+IIO_TRIGGER_EXAMPLE = n
```

Comment:
nitpick: I would remove these if the examples are not yet supported, to avoid confusion.